### PR TITLE
Image

### PR DESCRIPTION
**Pull request by JatuFaique**
Please review my Image component.
The Image component added.

### CODE REVIEW

File: image/image.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap");
+
+* {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```
Comment (khyatibaria) [inline image/image.css]: Suggestion reusable variables and colors can be kept in one file. eg: variables.css.

File: image/image.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./image.css">
+    <title>Image</title>
+</head>
+<body>
+    <h3>Images</h3>
+    <div class="image-section disp-flex">
+        <div class="image-bs">
+            <img class='image' src="https://picsum.photos/400" >
```
Comment (khyatibaria) [inline image/image.html]: Can add alt attribute for img tag

**Review by khyatibaria — CHANGES_REQUESTED** [review]
Rest of things looks good

File: image/image.css (hunk 1 above)
Comment (zabihshaik) [reply image/image.css]: Same suggestion as Khyati. Better to use reusable colors as variables and make it a separate css file

File: image/image.html (hunk 2 above)
Comment (zabihshaik) [inline image/image.html]: Better to add alt attribute to img tag

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just added a few comments. Please check.

**Pull request merged**